src/io/fasta.rs

```diff
@@ -3,14 +3,100 @@
 // This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! FASTA format reading and writing.
+//! Structs and trait to read and write file in FASTA format.
 //!
 //! # Example
 //!
+//! ## Read
+//!
+//! In this example we parse fasta file from stdin and compute some statistics
```

Comment:
In this example, we parse a fasta file from stdin and compute some statistics

Reply:
```suggestion
In this example, we parse a fasta file from stdin and compute some statistics
```